### ISSUE TITLE
Instructions for writing compatible manual zettels

### ISSUE THREAD

**Issue by melek** [srid/neuron#377]
A 'template' zettel/note with all the possible frontmatter and full filename rules & restrictions would be very helpful for when I'm on a system/platform where I must manually add/edit Zettels. Perhaps this exists, but I can't find something like that anywhere in the guide. Looking around I think it is something like this:

```
Filename: 00000001.md 
Note: Filenames must begin with the 8-character ID and end in .md, but can have any amount of legal characters after that.
File Contents:
---
date: 2020-09-06 10:27
title: This is a note title.
tags: 
 - tag1
 - tag2
---

# This is a note title (Neuron falls back to this if title is omitted from frontmatter)

This is the note text, including any subtitles. Branching is done with links to IDs in [[trible brackets]]. Normal (non-branching) links are done with a [[[double bracket]]], or as regular markdown links.
```

Is there such a template, or might there be room for such a template/manual zettel creation tutorial in the Guide?

**Comment by srid** [comment]
If I understand you correctly, you are asking for this feature: #328

Yes?

**Comment by melek** [comment]
A template is the core of the idea, however I'm also imagining a simple page in the guide - something along the lines of 'Manually adding to the Zettelkasten' for when Neuron is unavailable/not yet installed. 

My inspiration is that I've had trouble getting neuron to work on my machine, but want to convert my notes into a compatible format and manually upkeep them that way so when I finally am able to use Neuron it will work :)

**Comment by srid** [comment]
The markdown files in the neuron Zettelkasten don't have to be special. For example, this is a valid Zettel:

```
echo 'Hello world' > hello.md
```

So you don't have absolutely have to use `neuron new` to create a new zettel. Just create an empty file, and start writing. Maybe this can be explicitly stated in the guide ...

**Comment by melek** [comment]
That would be a great start! Does the first 'word' of a filename constitute the ID, then? And adding the front-matter helps with Neuron's filtering features, right?

**Comment by srid** [comment]
- ID is always the filename (with the `.md` extension). 
- Title is derived from the level-1 heading (eg: `# First heading`), or if there is no heading, then the first few letters of the first paragraph, unless you explicitliy specify the `title` key in metadata.
- Tags can be specified inline like you do on Twitter (eg: `#tag1`), as well as on the metadata key `tags` (or `keywords`)
- Date can be specified only in the metadata block, using the `date` key.

So technicalliy the front-matter is strictly necessarly only for specifying the date.

**Comment by melek** [comment]
Your explanation here is very helpful.

My current plaintext notes use an `Unique ID - Date Stamp - Title.md` format, where I sometimes update the title. Will linking `[[Unique ID]]` automatically link to the correct note in the same directory?

Apart from that question, I think it is up to you if you want to add this to the guide or just close and point people to this issue. Thanks for thinking about my question!

**Comment by srid** [comment]
@melek No, it won't do any such thing. It has to be exactly `[[Unique ID - Date Stamp - Title]]` if that is what your filename is.

If you prefer to have title in the ID, you can of course ignore the unique ID and timestamp in your filename, and then just use `[[Title]]`.

By the way, regular Markdown links are supported too.
